api/src/main/java/javax/json/Json.java

```diff
@@ -81,7 +81,8 @@
  * threads.
  */
 public final class Json {
-
+	static final JsonProvider PROVIDER = JsonProvider.provider();
```

Comment:
Currently this would actually decrease performance with default implementation if JsonProvider is cached by client application (as it is supposed to). See https://github.com/javaee/jsonp/blob/134135684b5f7a4002811b9e579d5b32edc08d55/impl/src/main/java/org/glassfish/json/JsonProviderImpl.java#L69
When JsonProvider instance is cached like this buffer pool exists only once. Because it uses ConcurrentLinkedQueue it will decrease performance in multithreaded environment because of synchronization.

Reply:
I certainly see dangers making it final. That way for the lifetime of the Json class in a particular JVM it would remain the same even if the underlying provider may change (less likely without OSGi but possible for some implementations)